core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -407,6 +413,9 @@ object KafkaConfig {
   val AdvertisedListenersDoc = "Listeners to publish to ZooKeeper for clients to use, if different than the listeners above." +
   " In IaaS environments, this may need to be different from the interface to which the broker binds." +
   " If this is not set, the value for `listeners` will be used."
+  val ListenerSecurityProtocolMapDoc = "Mapping between listener names and security protocols. Key and value are separated by a colon " +
```

Comment:
Since this is the main documentation - I'd give a bit more background. "You may want to configure the same security protocol on two different ports or IPs - for example to separate external and internal traffic. To do that..."